api/src/migrations/migrations/2024-02-23T18:41:32.ts

```diff
@@ -0,0 +1,21 @@
+import { Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('friend')
+    .addColumn('user1_id', 'integer')
+    .addColumn('user2_id', 'integer')
```

Comment:
`userId`s must be a reference to an existing user:
`(col) => col.references('user.id').notNull().onDelete('cascade')`